src/main/java/uk/gov/justice/digital/delius/service/OffenderIdentifierService.java

```diff
@@ -57,7 +58,7 @@ public IDs updateNomsNumber(String crn, UpdateOffenderNomsNumber updateOffenderN
 
         if (!existingOffendersAlreadyWithNomsNumber.isEmpty()) {
             log.info("No need to Update NOMS number since an offender with the new noms number already exists");
-            return existingOffendersAlreadyWithNomsNumber.stream().map(OffenderTransformer::idsOf).collect(Collectors.toList());
+            throw new ConflictingRequestException(String.format("NOMS number %s is already assigned to %s", updateOffenderNomsNumber.getNomsNumber(), existingOffendersAlreadyWithNomsNumber.stream().map(Offender::getCrn).collect(Collectors.joining(","))));
```

Comment:
Slightly archaic control by exception, but this follows existing pattern.  Something that we may want to move to Result type responses in the future ...